mac

```diff
@@ -138,7 +138,7 @@ brew "hub"
 brew "imagemagick"
 
 # Testing
-brew "qt@5.5"
+brew "qt@5.5" if MacOS::Xcode.installed?
 
```

Comment:
Good call. That has to be a different conditional since the `Brewfile` doesn't support a `link` method. Updated.